applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1634,6 +1687,17 @@ should_require_pin(MailboxJObj) ->
         'false' -> kzd_voicemail_box:pin_required(MailboxJObj)
     end.
 
+-spec is_ff_rw_enabled(kz_json:object()) -> boolean().
+is_ff_rw_enabled(MailboxJObj) ->
+    case ?IS_FF_RW_ENABLED of
+        'true' -> kzd_voicemail_box:is_ff_rw_enabled(MailboxJObj);
```

Comment:
Looks as `kzd_voicemail_box` module is deprecated. And need use `kzd_vmboxes` in this PR.
@jamesaimonetti, @lazedo could you look.

Reply:
They need to be merged into one module; that can be taken care of at a future time.

Reply:
But new functionality should go in kzd_vmboxes, that is the module that tracks the schema.